modules/backup-filesystem/backup.go

```diff
@@ -24,15 +24,22 @@ import (
 	"github.com/weaviate/weaviate/usecases/monitoring"
 )
 
-func (m *Module) GetObject(ctx context.Context, backupID, key string) ([]byte, error) {
-	metaPath, err := m.getObjectPath(ctx, backupID, key)
+func (m *Module) GetObject(ctx context.Context, backupID, key, bucketName, bucketPath string) ([]byte, error) {
+	var metaPath string
+	var err error
+	if bucketPath != "" {
+		metaPath, err = m.getObjectPath(ctx, bucketPath, backupID, key)
+	} else {
+		metaPath, err = m.getObjectPath(ctx, m.backupsPath, backupID, key)
+	}
+
 	if err != nil {
 		return nil, err
 	}
 
 	contents, err := os.ReadFile(metaPath)
 	if err != nil {
-		return nil, backup.NewErrInternal(errors.Wrapf(err, "get object '%s'", metaPath))
+		return nil, backup.NewErrInternal(errors.Wrapf(err, "1 get object '%s'", metaPath))
```

Comment:
can we remove `1` from the error message?

Reply:
can we have unique error messages?

Reply:
yes, but not by adding `1` at the beginning of the error message, I think that this change should be done in a separate PR

Reply:
opportunistic refactoring https://martinfowler.com/bliki/OpportunisticRefactoring.html